modules/Bio/EnsEMBL/Compara/RunnableDB/ProteinTrees/HomologyIDMapping.pm

```diff
@@ -176,9 +181,15 @@ sub _fetch_and_map_previous_homologies_from_file {
 sub write_output {
 	my $self = shift;
```

Comment:
```suggestion
    my $self = shift;
```
Since you are changing all the function's content already, fixing the indentation of this line won't hurt :)